### ISSUE TITLE
Video volume set before play is not applied on the video

### ISSUE THREAD

**Issue by tito** [kivy/kivy#3759]
Depending the core provider, setting a video volume before play() doesn't work:

- `video_ffmpeg`: volume is not saved internally, and volume is applied only if the internal player is created. It fail silently if the internal player is not yet created (only after play()).
- `video_ffpyplayer`: volume is saved internally, and applied to the internal player if created. The volume is not applied when the internal player is created.
- `video_gi`: volume is set internally and on the internal player. The internal player is created when the instance is created, so there is no issue.
- `video_gstplayer`: volume is set internally and on the internal player if created. The volume is also set when play(). There is no issue.
- `video_pyglet`: volume is set if the internal player is created. The internal player is created on load(), so no issue.

In conclusion: only `video_ffmpeg` and `video_ffpyplayer` are impacted.

**Comment by tito** [comment]
Well, ffpyplayer doesn't do live volume changes :/
